driver-core/src/main/java/com/datastax/driver/core/DataTypeCqlNameParser.java

```diff
@@ -65,6 +65,8 @@
                     .put("timeuuid", timeuuid())
                     .put("tinyint", tinyint())
                     .put("smallint", smallint())
+                    // duration is not really a native CQL type, but appears as so in system tables
+                    .put("duration", duration())
```

Comment:
Nice, I remember that it was a similar story with UDTs being a custom type between a later version of 2.0 and 2.1 (with protocol v3).  I'm glad that is the case.